Lumpen Radio/Radio.swift

```diff
@@ -37,6 +38,7 @@ class Radio: ObservableObject {
     
     // The audio player object
     private var player: AVPlayer?
+    private var playerItem: AVPlayerItem?
```

Comment:
Since the `playerItem` will reside in `player`, do we need it as a member attribute?

If we're concerned about it being cleaned for whatever reason from within Player, we can place a nil check in `setupNowPlaying()` against the item in `player` and reconstruct it there if needed.

Reply:
yeah good point. Removed it